pkg/mock/static/static.go

```diff
@@ -24,8 +24,9 @@ import (
 )
 
 var (
-	supportedPaths = make(map[string]interface{})
-	response       interface{}
+	supportedPaths            = make(map[string]interface{})
+	response                  interface{}
+	elasticInferenceAssocPath = "/latest/meta-data/elastic-inference/associations/eia-bfa21c7904f64a82a21b9f4540169ce1"
```

Comment:
should this be a const?

Reply:
on second thought, should we make this more dynamic by making it a hash like `map[string]bool` for text-response paths? It would match more closely with the way you set it up in metadata.go to be future-thinking

Reply:
- yup const would be better
- can you explain this a bit more? Would all paths be in this hash? Not sure if this would work given there are 3 types of responses now: text, json, json-formatted-but-text

Reply:
It looked like you set up a map in metadata.go so that if there are other nested structs in the future we can just add them to that map. Otherwise I thought you'd just hardcode the elastic inference stuff. Maybe I interpreted that wrong. 

Anyway, I'm suggesting doing something similar here in case in the future there are other paths that need to use `FormatJSONAndReturnTextResponse`

So doing something like:
```	
var (
    jsonTextResponse = map[string]bool{}{
        "/latest/meta-data/elastic-inference/associations/eia-bfa21c7904f64a82a21b9f4540169ce1" : true,
    }
)
// further down...

if _, ok := jsonTextRespose[req.URL.Path]; ok {
    server.FormatJSONAndReturnTextResponse(res, response)
} else {
    server.FormatAndReturnJSONResponse(res, response)
}
```

maybe unnecessary though 🙃

Reply:
ohhh I gotcha! Yeah you're right that'd be better. Addressing in next revision